test/test.ts

```diff
@@ -322,6 +322,18 @@ describe('gcs-resumable-upload', () => {
         };
         up.emit('writing');
       });
+
+      it('should save the uri to config', done => {
```

Comment:
@stephenplusplus what is the difference between this test and the test below? Are there tests for both cases (case where uri should be saved and case where uri should not be saved)? I might be missing something

Reply:
That's funny, I didn't realize I could have copy/pasted the whole thing 😆 The only thing that's changing in the tests is what triggers the call to `createURI()`.

`createURI()` won't call `set()` anymore, so I removed the test that used to confirm we called it. In these new tests, we're checking that where we call `createURI()` internally, we also call `set()` right after with the result.

Reply:
ah, makes sense!!